File: app/src/main/java/com/leeeyou/service/subscriber/DefaultHttpResultSubscriber.java
package com.leeeyou.service.subscriber;

import timber.log.Timber;

public abstract class DefaultHttpResultSubscriber<T> extends HttpResultSubscriber<T> {
    @Override
    public void _onError(int status, String msg) {

    }

    @Override
    public void onCompleted() {
        Timber.d(" --> onCompleted");
    }
}

File: app/src/main/java/com/leeeyou/service/subscriber/DefaultHttpResultSubscriber.kt
package com.leeeyou.service.subscriber

import timber.log.Timber

abstract class DefaultHttpResultSubscriber<T> : HttpResultSubscriber<T>() {
    override fun _onError(status: Int, msg: String?) {
        
    }

    override fun onCompleted() {
        Timber.d(" --> onCompleted")
    }
}
